locales/en/homepage.json

```
@@ -36,6 +36,9 @@
   "reuses.tosdr.author": "ToS;DR",
   "reuses.tosdr.subtitle": "Lets users collaboratively rate how respectful terms of services are of their rights.",
   "reuses.tosdr.title": "Terms of Service; Didn’t Read",
+  "reuses.report-lab-lex.author": "Lab-LEX",
+  "reuses.report-lab-lex.title": "Governance and regulation of social media",
```

That is the name of the research program, not of the paper itself.
```suggestion
  "reuses.report-lab-lex.title": "The evolution of social media terms of use and their impact on human rights",
```

If that is too long, then this shortened version could be used:
```suggestion
  "reuses.report-lab-lex.title": "Research on the impact on human rights",
```